packages/flame/lib/src/math/random_fallback.dart

```diff
@@ -0,0 +1,5 @@
+import 'dart:math';
+
+/// When you don't care about what [Random] object you have and don't want to
+/// create an unnecessary object you can use this pre-created object.
+final Random randomFallback = Random();
```

Comment:
is this just to avoid instantiating a random? I feel `?? Random()` is actually shorter than `?? randomFallback`

Reply:
Exactly, so that there aren't any new `Random` objects created in tight loops